src/com/amila/qamp/oopzadaca2/Task3/Robot/Robot.java

```diff
@@ -0,0 +1,101 @@
+package com.amila.qamp.oopzadaca2.Task3.Robot;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Robot {
+
+    // atributes
+    private int positionX;
+    private int positionY;
+    private int moves;
+    List<String> positions = new ArrayList<>();
```

Comment:
The code works, but it revolves around the format of how we print the output (x,y).

You are storing the positions as a List of String, but think about it, we are using OOP, is there a something better to store a list of Positions. Take inspiration in the classes Bookshelf and Classroom. Bookshelf stores Books, what could the Robot store.